docs/source/getting-started/installation.rst

```diff
@@ -38,10 +38,10 @@ To install the SDK on Mac or Linux open a terminal and run:
 The installer will setup the ``PATH`` variable for you. In order for it to take effect, you will have to
 log out and log in again.
 
-Installing the Enterprise Edition
+Installing Daml Enterprise
 *********************************
```

Comment:
```suggestion
Installing Daml Enterprise
**************************
```
I believe the linter will complain about the title and the `***` not having the same size.